src/meson.build

```diff
@@ -55,7 +55,7 @@ endif
 # Maintain compatibility with autotools; see: https://github.com/anholt/libepoxy/issues/108
 darwin_versions = [1, '1.0']
 
-epoxy_deps = [ dl_dep, ]
+epoxy_deps = [ dl_dep, egl_dep ]
```

Comment:
This isn't right. It causes EGL to always get added to `Requires.private` in pkgconfig even if it was not requested (but happened to be found). The library is loaded dynamically so it's not a dep.

Reply:
I reverted the change with commit 47d975d726ece5747333ccd9c96d2d76ac753924.